backend/app/Http/Resources/VisitorsFlowResource.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Http\Resources;
+
+use App\Contracts\ApiResponse;
+use App\DataTransformer\VisitorsFlow\ParameterFlowItem;
+use App\DataTransformer\VisitorsFlow\ParameterItem;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+use Illuminate\Support\Collection;
+
+class VisitorsFlowResource extends ResourceCollection implements ApiResponse
+{
+    public function toArray($request): array
+    {
+        return $this->presentCollection($this->collection);
+    }
+
+    public function presentParameterViews(ParameterItem $item): array
+    {
+        return [
+            'title' => $item->getTitle(),
+            'count' => $item->getCount(),
+        ];
+    }
+
+    public function presentVisitorsFlow(ParameterFlowItem $item): array
+    {
+        return [
+            'target_url' => $item->getTargetUrl(),
+            'level' => $item->getLevel(),
+            'views' => $item->getViews(),
+            'exit_count' => $item->getExitCount(),
+            'source_url' => $item->getSourceUrl()
+        ];
+    }
+
+    public function presentCollection(Collection $collection): array
+    {
+        return $collection
+            ->map(
+                function (Collection $items, $key) {
+                    switch ($key) {
```

Comment:
if you have a switch case inside presenter it means it should be two different presenters, if you have two presenters it means you should have two endpoints